apps/projects/app/components/Card/Revoked.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, EmptyStateCard, GU, Text, useTheme } from '@aragon/ui'
+import { EmptyWrapper } from '../Shared'
+import revoked from '../../assets/revoked.svg'
+
+const Illustration = () => <img src={revoked} alt="" />
+
+
```

Comment:
our linter doesn't complain about two line breaks in a row so I guess I have to 🙃